src/pages/AdvancedSearch/SearchForm/SearchForm.tsx

```diff
@@ -67,7 +85,14 @@ const SearchForm = (): JSX.Element => {
               name="query"
               rows={8}
             />
-            <Button type="submit">Search</Button>
+            <Button type="submit">{Label.SEARCH}</Button>
+            <Button
+              type="button"
+              onClick={() => copyToClipboard(crossModelQueryResultsJSON)}
+              disabled={!isCrossModelQueryLoaded || isCrossModelQueryLoading}
```

Comment:
I think we should also disable this if the result is an error.

Reply:
Makes sense! Added in the latest commit.